Automattic/wp-calypso
# Enable connection flow in Jetpack cloud

### Changes proposed in this Pull Request

This is a proof of concept to enable the connection flow in Jetpack cloud.

### Testing instructions

- Download this PR
- Run Calypso and Jetpack cloud concurrently
- Apply this patch to your sandbox and make `jetpack.wordpress.com` point to it (you'll need to change the port to `3000` if you're not running both platforms concurrently): D53031-code
- Select a self-hosted Jetpack site that's not connected yet
- Visit `/jetpack/connect/` and follow the steps to connect your site
- Verify that through the process you're staying on Jetpack cloud
